### ISSUE TITLE
Inaccessible Fake Radio Buttons

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/quizful#38]
## Problematic Code
https://github.com/Thinkful-Ed/quizful/blob/05508edae8570379f5e8b1d278ccc2a06c754fd1/src/components/answers.jsx#L30

## The Issue
`<li onClick={this.qCheck}>` is not focusable or mapped to the accessibility tree and is only accessible to sighted mouse users. 

```js
return <li onClick={this.qCheck}><i className={this.state.class} aria-hidden="true"></i><p>{this.props.name}</p></li>
```

## The Solution
Do not add `onClick` to non-interactive elements like `<div>`, `<li>` that are not mapped to the accessibility tree. Use `<button>`, `<a>`, form elements and/or ARIA instead to meet WCAG success criteria. 

Here we should probably be using `<fieldset>` and `<input type="radio">` or mimicking such usage with ARIA.

## See Also
 - [Twitter Thread](https://twitter.com/marcysutton/status/848305312587169792)
 - https://github.com/Thinkful-Ed/quizful/issues/35
